src/modules/ConsumerGroups/components/ConsumerGroupDetail/ConsumerGroupDetail.tsx

```diff
@@ -154,7 +154,7 @@ const ConsumerGroupDetail: React.FunctionComponent<
             <Text component={TextVariants.h4}>{t('consumerGroup.state')}</Text>
             <Text component={TextVariants.p}>
               <Text component={TextVariants.h2}>
-                {State[consumerGroupDetail?.state]}
+                {ConsumerGroupState[consumerGroupDetail?.state as keyof typeof ConsumerGroupState]}
```

Comment:
Can we figure out a better typing for both the `ConsumerGroupState` and the `ConsumerGroupDetail` so to avoid casting?

Reply:
Yes I want to fix just the types as why are now, deploy this PR on stage, make sure that nothing breaks on the test suite we have running against stage and then do a new pr with the updated sdks. This way if something breaks it should be easier to spot the culprit

Reply:
Good plan